AudioSensei/ViewModels/MainWindowViewModel.cs

```diff
@@ -741,11 +750,13 @@ private void CancelPlaylistCreation()
             PlaylistDescription = "";
         }
 
+        [NotNull]
```

Comment:
These void things are weird, we should make sure it won't be a null, right? right? right?

Reply:
No idea how did that happen
```suggestion
```